scriptoocloud/src/test/java/edu/asu/diging/scriptoocloud/core/service/impl/JgitServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package edu.asu.diging.scriptoocloud.core.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.net.MalformedURLException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import edu.asu.diging.scriptoocloud.core.data.GitRepositoryRepository;
+import edu.asu.diging.scriptoocloud.core.exceptions.InvalidGitUrlException;
+import edu.asu.diging.scriptoocloud.core.service.DeleteFilesService;
+
+/*
+ * 
+ * 
+ * @author Jason Ormsby
+*/
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { AnnotationConfigContextLoader.class})
+@TestPropertySource("classpath:config.properties")
+class JgitServiceImplTest {
+
+    @Value("${testPath}")
+    private String path;
+
+    @Mock
+    private DeleteFilesService deleteFilesService;
+
+    @InjectMocks
+    private JgitServiceImpl serviceToTest;
+    
+    @BeforeEach
+    public void init(){
+        MockitoAnnotations.initMocks(this);  
+    }
+    
+    @Test
+    public void test_clone_success(){
+
+        String folderName = path + "clone_success";
+        File file = new File(folderName);
+        
+        Assertions.assertDoesNotThrow(()->serviceToTest.clone(folderName, "https://github.com/diging/scriptoocloud"));
+      
+        Assertions.assertTrue(file.exists());
+        
+        DeleteFilesServiceImpl deleteFilesService = new DeleteFilesServiceImpl();
+        deleteFilesService.deleteDirectoryContents(file);
```

Comment:
hm, I understand why you do this for convenience, but I would just add a method to this class to delete a folder. If someone changes DeleteFilesServiceImpl and introduces bugs, not just the tests for DeleteFilesServiceImpl would fail, but also these test cases here, which would make it more confusing to figure out what is wrong.